pages/news.php

```diff
@@ -142,7 +142,7 @@ function news() {
 
 	if(!$page) {
 		if($player->hasAdminPanel()) {
-			echo "<p style='text-align:center;'><a href='$self_link?page=create_post'>New post</a></p>";
+			echo "<p style='text-align:center;'><a href='$self_link&page=create_post'>New post</a></p>";
```

Comment:
We finally have a better way to do this, added recently:
```suggestion
			echo "<p style='text-align:center;'><a href='<?= $system->router->getUrl('news', ['page' => 'create_post']) ?>'>New post</a></p>";
```